src/bin/casc.rs

```diff
@@ -3,23 +3,25 @@
 use selinux_cascade::compile_system_policy;
 use selinux_cascade::error::HLLErrorItem;
 
-use std::env;
+use clap::{App, Arg};
 use std::fs::File;
 use std::io::{Error, ErrorKind, Write};
 
-fn usage() {
-    println!("casc policy.cas");
-}
+const VERSION: &str = env!("CARGO_PKG_VERSION");
 
 fn main() -> std::io::Result<()> {
-    // TODO: Move all this to a parse args function
-    let args: Vec<String> = env::args().collect();
-    if args.len() != 2 {
-        usage();
-        return Err(Error::new(ErrorKind::InvalidInput, "Missing policy file"));
-    }
+    let matches = App::new("casc")
+        .version(VERSION)
+        .author("Daniel Burgener <dburgener@linux.microsoft.com>")
```

Comment:
You can use the CARGO_PKG_AUTHORS env as well.

Reply:
Looks like clap offers a crate_authors!() convencience macro to handle the formatting, so I switched to that.  Thanks!

Reply:
Hmm, there is also `crate_version!()` :)

Reply:
Good catch!  Let me submit a PR for that.